src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">    
+    <Import Project="..\..\build\common.props" />
+    <PropertyGroup>
+        <Description>Build-time references required to enable Razor view compilation as part of building the application.</Description>
+        <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
+        <RuntimeIdentifier Condition="!$(TargetFramework.StartsWith('netcoreapp'))">win7-x64</RuntimeIdentifier>
+        <PackageTags>cshtml;razor;compilation;precompilation;aspnetcore</PackageTags>
+        <PreserveCompilationContext>true</PreserveCompilationContext>
+        <OutputType>exe</OutputType>
+
+        <!-- Include the build outputs in the build directory (and not the lib directory) -->
+        <BuildOutputTargetFolder>build</BuildOutputTargetFolder>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Compile Include="**\*.cs" />
+        <None Include="build/**/*.*" Pack="true" PackagePath="build/%(RecursiveDir)%(FileName)%(Extension)" />
```

Comment:
Can be `Include="build\**\*"` in case there are extensionless files. Also, MSBuild convention is backslashes instead of forwardslashes (even on xplat).

Also, you can simplify to `PackagePath="%(Identity)"`